qa/tasks/cephfs/test_volumes.py

```diff
@@ -1193,3 +1193,30 @@ def test_mgr_eviction(self):
         mgr = self.mgr_cluster.get_active_id()
         self.mgr_cluster.mgr_fail(mgr)
         self.wait_until_evicted(sessions[0]['id'])
+
+    def test_subvolume_upgrade(self):
+        """
+        poor man's upgrade test -- rather than going through a full upgrade cycle,
+        emulate a subvolume by going through the wormhole and verify access to the
+        subvolume.
+        """
+
+        subvolume = self._generate_random_subvolume_name()
+
+        # emulate a old-fashioned subvolume
+        createpath = os.path.join(".", "volumes", "_nogroup", subvolume)
+        self.mount_a.run_shell(['mkdir', '-p', createpath])
```

Comment:
Let's have one for a subvolume with a real group name too!

Reply:
ACK